Add models around BAM

ttung opened the pull request:
### Description
AlignRead -> Bam -> CallConsensus

#### Issue
[ch64743](https://app.clubhouse.io/genepi/stories/space/64743)

### Test plan
none

phoenixAja commented on src/py/aspen/database/models/__init__.py:
```diff
@@ -1,4 +1,5 @@
 from .accessions import Accession, PublicRepository  # noqa: F401
+from .align_read import AlignRead, Bam  # noqa: F401
```
did you also want to import `CallConsensus`? I didn't see it below

ttung replied:
Yup, I do.

ttung replied:
Fixed.

Commit:
Add models around BAM

### Description
AlignRead -> Bam -> CallConsensus

#### Issue
[ch64743](https://app.clubhouse.io/genepi/stories/space/64743)

### Test plan
none
